ui/src/layout/Navigation.tsx

```diff
@@ -59,6 +66,13 @@ class Navigation extends Component<
                           to={'/messages/' + app.id}
                           key={app.id}>
                           <ListItem button>
+                              <ListItemAvatar style={{minWidth: 42}}>
+                                  <Avatar
+                                      style={{width: 32, height: 32}}
```

Comment:
I've changed the sizes a little bit so that it doesn't increase the height of the ListItem.